### ISSUE TITLE
Doesn't work with Siri

### ISSUE THREAD

**Issue by SwiftWinds** [LorenzoPane/browserdefault#38]
This tweak doesn't open in the specified browser after clicking a Siri search result.

[Bug demo](https://photos.app.goo.gl/RdSYcBEfi3FbKtTi8)

Thanks for this great tweak, and I'd be eternally greatful if this was fixed! :)

**Comment by LorenzoPane** [comment]
Thanks for the bug report. I likely won't fix it in browserdefault since I'm currently working on a replacement in my free time. I'll keep this functionality in mind, though. It's not in a state to be used yet, but you can track development [here](https://github.com/LorenzoPane/EvilScheme)

**Comment by SwiftWinds** [comment]
OK, that sounds awesome. Thanks for the quick response, and I look forward to seeing this project! 😄

**Comment by SwiftWinds** [comment]
One idea of an enhancement I have is--similar to #36--to blacklist certain URLs and apps from BrowserDefault (and now EvilScheme). For example, I have a shortcut to open a YouTube link in invidio.us, but [only Safari supports background audio for > 30 seconds](https://github.com/LorenzoPane/browserdefault/issues/36), so a fantastic feature would be to blacklist sites (e.g., https://invidio.us/*) from BrowserDefault/EvilScheme. :)

**Comment by LorenzoPane** [comment]
Yeah I've thought a bit about that. As of now, my [model](https://github.com/LorenzoPane/EvilScheme/blob/e7b8860499de949a3e4ebfcdaa812d11eccbf3a8/EvilKit/src/EVKAppAlternative.m#L20) uses regexes to differentiate between differing domain actions, so blacklisting a domain should be trivial for an end user. Additionally, most intercepted requests have a source bundle ID stored somewhere (I can't remember the key of the top of my head), so blacklisting a source application could be implemented. Since I hook at the FrontBoard level, disabling code substitution in an app as @Elehiggle tried wouldn't prevent my redirects.

**Comment by SwiftWinds** [comment]
Ahh, I see. Good to hear that it's relatively simple to implement! :)
